matrix-sdk/src/main/java/org/matrix/androidsdk/db/MXMediasCache.java

```diff
@@ -517,6 +534,47 @@ public void clearTmpCache() {
         }
     }
 
+    /**
+     * Move a decrypted media file to the /share folder, to avoid this file to be deleted if in the /tmp folder
+     *
+     * @param fileToMove The file to move
```

Comment:
note: `filename` parameter is not commented here

Reply:
done